src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
I vote exposing it only on the dimension object with _ as the prefix
On Fri, Mar 18, 2016 at 12:47 PM Douglas Armstrong <notifications@github.com>
wrote:

> In src/crossfilter.js
> <https://github.com/crossfilter/crossfilter/pull/32#discussion_r56695490>:
>
> > @@ -121,6 +138,11 @@ function crossfilter() {
> >      offset = tmp.offset;
> >      one = tmp.one;
> >      zero = ~one;
> > +
> > +    // Expose the dimension mask for internal use
> > +    dimension._offset = offset;
>
> Well, since map keys are coerced to strings we would need a unique string
> key for each dimension then. And since the offset/one/zero is defined in
> the dimension constructor itself it would also mean that map in the
> crossfilter scope would need to be exposed. So, it seemed cleaner to expose
> private offset/zero/one in the dimension then expose the private
> offset/zero/one map in the crossfilter scope...
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/crossfilter/crossfilter/pull/32/files/9884cc984fa733f5b9760f57317a5aba7369280e#r56695490>
>